CHANGELOG.md

```diff
@@ -5,6 +5,24 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [v1.12.0] - 2021-04-27
+
+### Added:
+
+- Add "cancel" icon SVG [#27](https://github.com/hypothesis/frontend-shared/pull/27)
+- Add shared base element styling [#33](https://github.com/hypothesis/frontend-shared/pull/33)
+- Add reset styles and mixins [#32](https://github.com/hypothesis/frontend-shared/pull/32)
+- Add link color to \_colors.scss [#34](https://github.com/hypothesis/frontend-shared/pull/34)
```

Comment:
This is probably not worthy of change-logging.

Reply:
fixed